<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableDisplay</class>
 <widget class="QWidget" name="TableDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="StudentBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Student</string>
   </property>
   <widget class="QTableView" name="StudentView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>191</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ExamBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>191</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>Exam</string>
   </property>
   <widget class="QTableView" name="ExamView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="WknsBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>621</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Weakness</string>
   </property>
   <widget class="QTableView" name="WknsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>621</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
